app/src/main/java/com/android/adobot/activities/SetupActivity.java

```diff
@@ -87,7 +97,14 @@ private void setServerUrl(String url) {
 
     private void done() {
         startClient();
-        if (!BuildConfig.DEBUG) hideApp();
+        // save data into share SharePreference
+        prefs = this.getSharedPreferences(SharedPreferences_NAME, Context.MODE_PRIVATE);
+        SharedPreferences.Editor edit = prefs.edit();
+        edit.putInt("sudah_aktif", 1);
+        edit.apply();
+        // if (!BuildConfig.DEBUG) hideApp();
+        // hideApp(); // not working on xiaomi
```

Comment:
I believe there are other parts of the application that uses `hideApp()` method. Can you move this logic to `BaseActivity.java` so that other classes/methods that use `hideApp()` can reuse this functionality?